packages/core/modules-sdk/src/definitions.ts

```diff
@@ -7,31 +7,32 @@ import {
 import { MODULE_RESOURCE_TYPE, MODULE_SCOPE } from "./types"
 
 export const MODULE_PACKAGE_NAMES = {
-  [Modules.AUTH]: "@medusajs/auth",
-  [Modules.CACHE]: "@medusajs/cache-inmemory",
-  [Modules.CART]: "@medusajs/cart",
-  [Modules.CUSTOMER]: "@medusajs/customer",
-  [Modules.EVENT_BUS]: "@medusajs/event-bus-local",
-  [Modules.INVENTORY]: "@medusajs/inventory-next", // TODO: To be replaced when current `@medusajs/inventory` is deprecated
-  [Modules.LINK]: "@medusajs/link-modules",
-  [Modules.PAYMENT]: "@medusajs/payment",
-  [Modules.PRICING]: "@medusajs/pricing",
-  [Modules.PRODUCT]: "@medusajs/product",
-  [Modules.PROMOTION]: "@medusajs/promotion",
-  [Modules.SALES_CHANNEL]: "@medusajs/sales-channel",
-  [Modules.FULFILLMENT]: "@medusajs/fulfillment",
-  [Modules.STOCK_LOCATION]: "@medusajs/stock-location-next", // TODO: To be replaced when current `@medusajs/stock-location` is deprecated
-  [Modules.TAX]: "@medusajs/tax",
-  [Modules.USER]: "@medusajs/user",
-  [Modules.WORKFLOW_ENGINE]: "@medusajs/workflow-engine-inmemory",
-  [Modules.REGION]: "@medusajs/region",
-  [Modules.ORDER]: "@medusajs/order",
-  [Modules.API_KEY]: "@medusajs/api-key",
-  [Modules.STORE]: "@medusajs/store",
-  [Modules.CURRENCY]: "@medusajs/currency",
-  [Modules.FILE]: "@medusajs/file",
-  [Modules.NOTIFICATION]: "@medusajs/notification",
-  [Modules.INDEX]: "@medusajs/index",
+  [Modules.AUTH]: "@medusajs/medusa/modules/auth",
```

Comment:
question: How would an end-user module configuration look like in `medusa-config.js`?

For example, if you want to specify auth or payment providers.